### PR TITLE
Gzip js

### PR DESCRIPTION
**Pull request by ayushgarg1804**
Wrapping up Gzipping Javascript resources and configuring production environment in webpack. Also includes middleware for minifying HTML.

### CODE REVIEW

File: helloWorld/settings.py [hunk 1]
```diff
@@ -46,6 +46,7 @@
 ]
 
 MIDDLEWARE_CLASSES = [
+    'htmlmin.middleware.HtmlMinifyMiddleware',
```
Comment (mbad0la) [inline helloWorld/settings.py:49]: This is not required

**Review by mbad0la — CHANGES_REQUESTED** [review]
Just a small change. Everything else looks good

File: helloWorld/settings.py (hunk 1 above)
Comment (ayushgarg1804) [reply helloWorld/settings.py:49]: Why is it not required? I know html size is quite low for now but I included it for making it scalable.

Comment (mbad0la) [reply helloWorld/settings.py:49]: As this is a React-mounted application and we aren't intending to implement server-side rendering, the HTML content we have atm is final. Hence, there isn't any need to scale.

But anyway, there is no impact of this plugin, so I'll merge this!

**Pull request merged**